modules/contact/test/tests/nodal_scaling/two_block_slide.i

```diff
@@ -0,0 +1,279 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  volumetric_locking_correction = true
+[]
+
+[Mesh]
+  [left_block]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = -1.0
+    xmax = 0.0
+    ymin = -0.6
+    ymax = 0.6
+    nx = 5
+    ny = 5
+    elem_type = QUAD4
+  []
+  [left_block_sidesets]
+    type = RenameBoundaryGenerator
+    input = left_block
+    old_boundary = '0 1 2 3'
+    new_boundary = '10 11 12 13'
+  []
+  [left_block_id]
+    type = SubdomainIDGenerator
+    input = left_block_sidesets
+    subdomain_id = 1
+  []
+
+
+  [right_block]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0.0
+    xmax = 1.0
+    ymin = -0.5
+    ymax = 0.5
+    nx = 5
+    ny = 5
+    elem_type = QUAD4
+  []
+  [right_block_sidesets]
+    type = RenameBoundaryGenerator
+    input = right_block
+    old_boundary = '0 1 2 3'
+    new_boundary = '20 21 22 23'
+  []
+  [right_block_id]
+    type = SubdomainIDGenerator
+    input = right_block_sidesets
+    subdomain_id = 2
+  []
+
+  [combined_mesh]
+    type = MeshCollectionGenerator
+    inputs = 'left_block_id right_block_id'
+  []
+
+  [left_lower]
+    type = LowerDBlockFromSidesetGenerator
+    input = combined_mesh
+    sidesets = '11'
+    new_block_id = '10001'
+    new_block_name = 'secondary_lower'
+  []
+  [right_lower]
+    type = LowerDBlockFromSidesetGenerator
+    input = left_lower
+    sidesets = '23'
+    new_block_id = '10000'
+    new_block_name = 'primary_lower'
+  []
+[]
+
+[Variables]
+  [./disp_x]
+    block = '1 2'
+  [../]
+  [./disp_y]
+    block = '1 2'
+  [../]
```

Comment:
No need to add_variables in the TM action and add them simultaneously. I fall for this often.

Reply:
Thanks @recuero ! I just realized that I added this work-in-progress test by accident. This should not appear in this PR :)  (I will make the change in my other local branch though!)